### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VVinnikov**
PR for README.MD on kafka-postgres connector deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,61 @@
...
+mvn install -DskipTests=true
 ```
 
-## How to run
-
-Default configuration can be found in each project src/main/resources/application.yml it requires PostgreSQL connection.
-Kafka configuration can be left as is (broker configuration coming with the request).
+### Connectors configuration
+The connector configuration files `application.yml` are located in the respective folders
+`kafka-postgres-writer/src/main/resources/` and kafka-postgres-reader/src/main/resources/`.
```
Comment (vbir94) [inline README.md]: missing quote before `reader` folder

Comment (VVinnikov) [reply README.md]: Added quote on the PR branch

File: README.md [hunk 2]
```diff
@@ -1,43 +1,61 @@
...
+sudo ln -s ~/kafka-postgres-connector/kafka-postrges-writer/src/main/resources/application.yml ~/kafka-postgres-connector/kafka-postrges-writer/target/application.yml
+sudo ln -s ~/kafka-postgres-connector/kafka-postrges-reader/src/main/resources/application.yml ~/kafka-postgres-connector/kafka-postrges-reader/target/application.yml
 ```
 
-After starting connector, available endpoints are:
+### Run services
+#### Run the kafka-postgres-writer connector as a single jar
+```shell script
+cd ~/kafka-postgres-connector/kafka-postgres-writer/target
+java -Dspring.profiles.active=default -jar kafka-postgres-writer-<version>.jar
```
Comment (FDoKE) [inline README.md]: no need in profile

File: README.md [hunk 3]
```diff
@@ -1,43 +1,61 @@
...
+```
+#### Run the kafka-postgres-reader connector as a single jar
+```shell script
+cd ~/kafka-postgres-connector/kafka-postgres-reader/target
+java -Dspring.profiles.active=default -jar kafka-postgres-reader-<version>.jar
```
Comment (FDoKE) [inline README.md]: no need in profile

File: README.md (hunk 2 above)
Comment (VVinnikov) [reply README.md]: Removed mention of profiles.active

File: README.md (hunk 3 above)
Comment (VVinnikov) [reply README.md]: Removed mentions of profiles.active

File: README.md [hunk 4]
```diff
@@ -1,43 +1,60 @@
...
 
+## Local deployment
+
+### The cloning and building of kafka-postgres-connector
 ```shell script
-mvn clean package
+#clone
+git clone https://github.com/arenadata/kafka-postgres-connector
+# build without any tests 
+cd ~/kafka-postgres-connector
```
Comment (artem-krutov) [inline README.md:15]: No need in the `~/` in the beginning. `git clone` clones repo to the current folder, and `~/` means the base folder for a user.

Comment (VVinnikov) [reply README.md:15]: Made cloning from repository into base folder for a user as:
`git clone https://github.com/arenadata/kafka-postgres-connector ~/`

Made the same way for cloning in all other README-files.

**Review by artem-krutov — APPROVED** [review]

**Pull request merged**